src/app/algo-practice/[category]/[level]/[problemId]/page.tsx

```
@@ -0,0 +1,156 @@
+type ParamsType = {
+  category: string;
+  level: 'beginner' | 'intermediate' | 'advanced';
+  problemId: string;
+};
+
+interface Props {
+  params: Promise<ParamsType>;
+}
+
+export default function ProblemSolvingPage({ params }: Props) {
+  const { category, level, problemId } = use(params);
```

ESLint

[error] 'problemId' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)